src/catradar/positions_updater.py

```diff
@@ -115,7 +116,7 @@ def update_positions(
         cursor_push(positions, cursor_pos)
 
     for i in range(N):
-        positions[i] += speed_mult * velocities[i]
+        positions[i] += speed_mult * velocities[i] * dt * 60
```

Comment:
nit: it's preferable to use a constant for FPS.

Reply:
But this is not FPS, it's just a random coefficient for good movement speed.